app/assets/stylesheets/modules/edit_form.scss

```diff
@@ -0,0 +1,40 @@
+@import "variables";
+@import "mixins";
+
+.form {
+
+  .legend{
+    font-family:$dogstay-font-theme;
+    font-size: 2em;
+  }
+
+  img {
+    @include image-style;
+    margin-top: 3em;
```

Comment:
Properties should be ordered height, margin-left, margin-right, margin-top, width